doc: make private location configuration pages more consistent between them, closes HYB-448 (#184)
gatling/frontline-cloud-doc: content/reference/admin/private_locations/configuration/azure/index.md

weight: 22053
---

## Permissions

Azure private locations require the control plane to have credentials configured in order to instantiate virtual machines and associated resources.

> Those can be set through environment variables in your control plane or via [Azure RBAC](https://learn.microsoft.com/en-us/azure/role-based-access-control/overview). Select the most appropriated method depending on your infrastructure, or check our [installation guide]({{< ref "../../installation/azure" >}})  for deployment with [Azure Container Apps](https://azure.microsoft.com/en-us/products/container-apps).

Reviewer: ```suggestion
Credentials can be set through environment variables in your control plane or via [Azure RBAC](https://learn.microsoft.com/en-us/azure/role-based-access-control/overview). Select the most appropriate method depending on your infrastructure, or check our [installation guide]({{< ref "../../installation/azure" >}})  for deployment with [Azure Container Apps](https://azure.microsoft.com/en-us/products/container-apps).
```

* [Client id and secret](https://learn.microsoft.com/en-us/answers/questions/834401/hi-i-want-my-client-id-and-client-secret-key)
* [Subscription id](https://learn.microsoft.com/en-us/azure/azure-portal/get-subscription-tenant-id)

## System requirements

Azure private locations rely on some dependencies.

> So when using a custom image, make sure following are available:

Reviewer: ```suggestion 
When using a custom image, make sure the following are available:
```

- [cloud-init](https://learn.microsoft.com/en-us/azure/virtual-machines/custom-data) integration.
- [jq](https://jqlang.github.io/jq/download/) a lightweight and flexible command-line JSON processor.
- [curl](https://curl.se/download.html) a command line tool and library for transferring data with URLs
- [Java runtime environment](https://openjdk.org/install/): OpenJDK 64bits LTS versions: 11, 17 or 21 (see [Gatling prerequisites](https://gatling.io/docs/gatling/tutorials/installation/#java-version))

{{< alert tip >}}

> Learn how to tune the OS for more performance, configure the open files limit, the kernel and the network [here](https://gatling.io/docs/gatling/reference/current/core/operations/).

Reviewer: ```suggestion
Learn how to tune the OS for more performance and configure the open files limit, the kernel, and the network [here](https://gatling.io/docs/gatling/reference/current/core/operations/).
```

> Learn how to tune the OS for more performance, configure the open files limit, the kernel and the network [here](https://gatling.io/docs/gatling/reference/current/core/operations/).

Reviewer: I usually avoid using "here" for a link in the documentation. You could replace "here" with "in the Documentation" and add the link to the word Documentation.
